src/Tizen.NUI/src/public/Events/LongPressGestureDetector.cs

```diff
@@ -277,6 +277,21 @@ public LongPressGesture LongPressGesture
                     longPressGesture = value;
                 }
             }
+
+            /// <summary>
+            /// Sets a value that indicates whether the event handler has completely handled the event or whether the system should continue its own processing.
+            /// It will be reset after the gesture callback is called.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public bool Handled
+            {
```

Comment:
Set only property라소 경고가 발생하는데, 이렇게 변수를 두는게 어떨까요?
```suggestion
            bool _handled;
            [EditorBrowsable(EditorBrowsableState.Never)]
            public bool Handled
            {
                   get => _handled;
                   set 
                   {
                       _handled = value;
                       Interop.Actor.SetNeedGesturePropagation(View.getCPtr(view), value);
                       if (NDalicPINVOKE.SWIGPendingException.Pending)
                           throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                   }
```

Reply:
넵 말씀해 주신것처럼 수정해 보았습니다.~